omod/src/main/resources/liquibase.xml

```diff
@@ -135,7 +141,7 @@
 
             select concept_map_type_id from concept_map_type where name='SAME-AS' into @concept_map_type_id;
 
-            call add_concept(@concept_id, @concept_name_short_id, @concept_name_full_id, 'BACTERIOLOGY CONCEPT SET', 'Bacteriology Concept Set', 'N/A', 'Misc', true);
+            call bacteriology_add_concept(@concept_id, @concept_name_short_id, @concept_name_full_id, 'BACTERIOLOGY CONCEPT SET', 'Bacteriology Concept Set', 'N/A', 'Misc', true);
```

Comment:
This change will cause the checksum of the changeset to change and result in error on a system where this migration/changeset has already run. 
You should either create a new changeset with 1) different ID (and ensure using precondition that you are doing idempotent migrations) or 2) override using "validCheckSum" option (not preferred). 
For option 1) You need to ensure idempotent migrations. If you go with this approach, I would suggest that you split up the migrations into multiple changesets with each having specific precondition checks for specific concept's existence.

Reply:
Oyaa nice catch!